src/coreComponents/physicsSolvers/solidMechanics/SolidMechanicsLagrangianFEM.cpp

```diff
@@ -146,7 +146,7 @@ void SolidMechanicsLagrangianFEM::registerDataOnMesh( Group & meshBodies )
     nodes.registerField< solidMechanics::incrementalDisplacement >( getName() ).
       reference().resizeDimension< 1 >( 3 );
 
-    if( m_timeIntegrationOption != TimeIntegrationOption::QuasiStatic )
+    if( m_timeIntegrationOption != TimeIntegrationOption::QuasiStatic || Group::hasGroupByPath("/Outputs/ChombIO") )
```

Comment:
We want to register these fields if there is a ChombIO output in the problem spec.
This is called after all the input has been parsed and the mesh created, so there should never be an issue with creation order, etc